### ISSUE TITLE
Unshare and Link shares are missing 15.0.2

### ISSUE THREAD

**Issue by IamAliBaba** [nextcloud/server#13656]
hello,

We upgrades to 15.0.2 and seems unshare and link sharings are missing

**Comment by BurtGummer** [comment]
this one?

![image](https://user-images.githubusercontent.com/1249295/51366740-5954ad80-1ae7-11e9-906e-ec5ba8fd5dd7.png)

**Comment by danieldietsch** [comment]
I have the same issue. And @BurtGummer , yes, this menu is missing.

**Comment by SuziQ407** [comment]
Hi, same issue to me -- NC-15.0.2, complete new installation (after upgrade failure).
Not only the menu is missing, the whole "Share link" line inclusive "Copy to clipboard". Tested in five browsers on Linux, also browsers without any Addon. Also tested in IE-11 on Win-7.

**Comment by BurtGummer** [comment]
The app "File sharing" is enabled and active?

**Comment by MysticalGrape** [comment]
I have the same issue on 15.02 stable. Browser is Firefox 64.
![image](https://user-images.githubusercontent.com/8400428/51425818-1c52fd00-1c46-11e9-8d6a-8296be9907e4.png)
Sharing is enabled (Allow users to share via link). I can still share/unshare folders and files in android mobile app.

Thanks guys for all you hard work.

**Comment by caveman99** [comment]
Same here. The functionality is there, i can still create and edit shares via the apps and the desktop client. On the Web GUI the controls are completely invisible.

**Comment by SuziQ407** [comment]
@BurtGummer, yes, "File sharing" v1.7.0 is (and was) already enabled. If I click on an existing share I can't neither disable it nor change some share settings.

**Comment by danieldietsch** [comment]
Yes, sharing is enabled and active.

**Comment by violoncelloCH** [comment]
is the app socialsharing_email active on your instances?
then it's probably a duplicate of nextcloud/socialsharing#29

**Comment by MysticalGrape** [comment]
I can confirm that disabling "Social sharing via email" app sorted out the issue for me. Thank you!
![image](https://user-images.githubusercontent.com/8400428/51434255-5a413700-1cc1-11e9-9d13-53037f138e3b.png)

**Comment by danieldietsch** [comment]
Indeed, it works again if I disable Social sharing via email.

**Comment by SuziQ407** [comment]
Only disabling the app "Social sharing via email" doesn't correct this issue, I had to disable the app "Share by mail" too. NOW the share menu appears correctly again. Many thanks for your support! (y)

**Comment by sysrich** [comment]
I had to remove the "Social sharing via email" and the "Social sharing via twitter" apps

The "Share by mail" app is present on both of my 15.0.2 instances with no issue

**Comment by r4sas** [comment]
Same issue on 15.0.2. Disabling e-mail share apps doesn't help.

**Comment by supremesyntax** [comment]
Disabling "Social sharing via Diaspora" helped here, too. 
It was the only social sharing app enabled in my instance.
